README.md

```diff
@@ -2,7 +2,7 @@
 
 [![auto-update](https://github.com/iptv-org/iptv/actions/workflows/auto-update.yml/badge.svg)](https://github.com/iptv-org/iptv/actions/workflows/auto-update.yml)
 
-Collection of publicly available IPTV (Internet Protocol television) channels from all over the world.
+Collection of publicly available IPTV (Internet Protocol television) channels from all over the world. See [FAQ](FAQ.md).
```

Comment:
```suggestion
Collection of publicly available IPTV (Internet Protocol television) channels from all over the world.
```

Reply:
This file is generated automatically. To change the readme.md you need to edit the [.readme/template.md](https://github.com/iptv-org/iptv/blob/master/.readme/template.md)

Reply:
And maybe it would be better to add a full-fledged section with a link under Resources. Something like this:

```
## FAQ

Answers to the most popular questions you can find in the [FAQ.md](FAQ.md).
```

Reply:
I forgot. Thanks for the suggestion.